README.md

```diff
@@ -0,0 +1,112 @@
+![sttp-model](https://github.com/softwaremill/sttp-openai/raw/master/banner.png) Banner should be changed to sttp openAi
+
+[![Join the chat at https://gitter.im/softwaremill/sttp](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/softwaremill/sttp)
+[![CI](https://github.com/softwaremill/sttp-openai/workflows/CI/badge.svg)](https://github.com/softwaremill/sttp-openai/actions?query=workflow%3ACI+branch%3Amaster)
+
+[//]: # ([![Maven Central]&#40;https://maven-badges.herokuapp.com/maven-central/com.softwaremill.sttp&#41;&#40;https://maven-badges.herokuapp.com/maven-central/com.softwaremill.sttp&#41;)
```

Comment:
what does `[//]: #` do here?

Reply:
It comments it out, since nothing is posted yet under sttp.openai in maven repo it cannot load svg and link is pointing to non existing resource.

We can just remove it for now if you wish and add it once we have it posted on mvn